constants.js

```diff
@@ -106,7 +106,7 @@ const constants = {
     // user metadata header to set object locationConstraint
     objectLocationConstraintHeader: 'x-amz-meta-scal-location-constraint',
     // eslint-disable-next-line camelcase
-    externalBackends: { aws_s3: true, azure: true },
+    externalBackends: { aws_s3: true, azure: true, gcp: true },
```

Comment:
Why not call it google storage or `gs` ?

Reply:
We went with gcp (Google Cloud Platform) because it reminded us of aws (Amazon Web Services).